src/components/cardbuilder/cardBuilder.js

```diff
@@ -1547,6 +1547,10 @@ import 'programStyles';
                     return '<span class="cardImageIcon material-icons book"></span>';
                 case 'Folder':
                     return '<span class="cardImageIcon material-icons folder"></span>';
+                case 'BoxSet':
+                    return '<span class="cardImageIcon material-icons collections"></span>';
+                case 'Playlist':
+                    return '<span class="cardImageIcon material-icons queue_music"></span>';
```

Comment:
Actually it could also be a video playlist. But music is used more often.

Reply:
@dmitrylyzo Didn't found a more appropriate icon for it sadly. The queue or playlist ones doesn't look like related to playlists.

I checked material-icons.io for reference

Reply:
This one could fit: https://material.io/resources/icons/?icon=queue&style=baseline

Reply:
![list-2](https://user-images.githubusercontent.com/56478732/86450355-0f36aa80-bd22-11ea-8449-3c037bbfa0e6.png)

Reply:
Nice, view_list would be perfect :)

It stays consistent as well.

Reply:
I agree with the view list icon.

Reply:
@dkanada @MrTimscampi @dmitrylyzo Best one imo as well, so done!